app/src/main/java/com/babylon/wallet/android/presentation/wallet/WalletViewModel.kt

```diff
@@ -91,9 +88,11 @@ class WalletViewModel @Inject constructor(
 
     private val refreshFlow = MutableSharedFlow<Unit>()
     private val accountsFlow = combine(
+        getEntitiesWithSecurityPromptUseCase(),
```

Comment:
As per our discussion, the accounts flow could stay independent of security prompts flow for a faster list displaying